bears/scss/SCSSLintBear.py

```diff
@@ -13,12 +16,295 @@ class SCSSLintBear:
     def create_arguments(filename, file, config_file):
-        return filename,
+        return filename, '--config', config_file
+
+    @staticmethod
+    def generate_config(filename, file,
+                        space_before_bang: bool=True,
+                        space_after_bang: bool=False,
+                        allow_chained_classes: bool=False,
+                        allow_color_keywords: bool=False,
+                        use_color_variables: bool=True,
+                        allow_debug_statement: bool=False,
+                        check_declaration_order: bool=True,
+                        allow_duplicate_properties: bool=False,
+                        allow_consecutives_duplicate_property: bool=False,
+                        else_on_same_line: bool=True,
+                        force_empty_line_between_blocks: bool=True,
+                        allow_empty_rules: bool=False,
+                        use_short_hexadecimal_length_style: bool=True,
+                        use_lowercase_hexadecimal: bool=True,
+                        allow_hexadecimal_validation: bool=True,
+                        allow_id_selector: bool=False,
+                        allow_important_rule_in_properties: bool=False,
+                        use_spaces: bool=True,
+                        indent_size: int=2,
+                        leading_zero: str='exclude_zero',
+                        allow_mergeable_selectors: bool=False,
+                        allow_leading_underscore: bool=True,
+                        function_naming_convention: str='hyphen',
+                        mixin_naming_convention: str='hyphen',
+                        variable_naming_convention: str='hyphen',
+                        placeholder_naming_convention: str='hyphen',
+                        max_depth: int=3,
+                        use_placeholder_selector_in_extend: bool=True,
+                        max_properties: int=10,
+                        allow_unit_on_zero_values: bool=False,
+                        check_ulrs_format: bool=True,
+                        urls_in_quotes: bool=True,
+                        allow_unnecesseary_parent_reference: bool=False,
+                        allow_unnecessary_mantissa: bool=False,
+                        allow_trailing_whitespaces: bool=False,
+                        allow_trailing_semicolon: bool=True,
+                        check_imports_path: bool=True,
+                        allow_filename_leading_underscore: bool=False,
+                        allow_extension: bool=False,
+                        use_length_variables: bool=True,
+                        check_properties_spelling: bool=True,
+                        extra_properties: list=(),
+                        disabled_properties: list=(),
+                        check_pseudo_elements: bool=True,
+                        space_between_parentheses: int=0,
+                        spaces_around_operators: str=1):
```

Comment:
In other configs it says space, here it says spaces. Is this the config name being used in other bears?